highScoreWord/solution.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+function high(x){
+  let arr = x.split(" ");
+  let newArr = arr
+    .map(x => x.split("")

```

Comment:
forEach() would be more appropriate to use than .map() for this kata.
It works, but like we said yesterday, this kata's challenge was not to transform data but to evaluate data.

If you ever have time, you should give it a try using forEach. :)